src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -7591,6 +7592,7 @@ pbsv1mod_meth_is_attrib_val_settable(PyObject *self, PyObject *args, PyObject *k
 			break;
 
 		case HOOK_EVENT_EXECJOB_ATTACH:
+		case HOOK_EVENT_EXECJOB_CONFIGURE:
```

Comment:
The message refers to an execjob_attach hook, but could also be execjob_configure. I suggest you use a different (but similar) message for the execjob_configure case.

Reply:
Oh yes, I missed that. I'll fix.